_posts/2021-01-31-choosing-randomization-unit.md

```diff
@@ -0,0 +1,353 @@
+---
+layout: post
+title: Choosing your randomization unit in online A/B tests
+author: ianwhitestone
+summary: A discussion around the implications of different randomization units in online controlled experiments
+comments: true
+image: images/zappa-serverless-docker/main.png
+---
+
+{% include head.html %}
+
+Despite being considered the gold standard approach for determining the true effect of a change, there are still many statistical traps & biases that can spring up when A/B tests are not set up correctly. This post will discuss some of the nuances you must consider when setting up your A/B test. In particular, the randomization unit you choose, and the corresponding metrics.
```

Comment:
```suggestion
Despite being considered the gold standard approach for determining the true effect of a change, there are still many statistical traps & biases that can spring up when A/B tests are not set up correctly. This post will discuss some of the nuances you must consider when choosing a randomization unit for your A/B test.
```

if you drop metrics stuff. even if you don't, consider combining the two sentences.